.storybook/stories/changelog.stories.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta, Description } from '@storybook/addon-docs';
+
+import changelog from '../../CHANGELOG.md';
+
+<Meta title="Changelog" />
```

Comment:
Can you use the constant here? The sorting wouldn't work if these were different
```suggestion
<Meta title={CHANGELOG} />
```

Reply:
Ah, you're right, but the constants are also not used in the other mdx files, I'll create a follow-up for it if you don't mind :)